packages/react-composites/src/composites/common/icons.tsx

```diff
@@ -53,6 +60,12 @@ export const COMPOSITE_ONLY_ICONS = {
   /* @conditional-compile-remove-from(stable) Chat_Notification_Icon*/
   ControlBarButtonBadgeIcon: <Circle20Filled />,
   /* @conditional-compile-remove-from(stable): meeting/calling-composite */
+  ControlBarChatButtonActive: <Chat20Filled />,
+  /* @conditional-compile-remove-from(stable): meeting/calling-composite */
+  ControlBarChatButtonInactive: <Chat20Regular />,
+  /* @conditional-compile-remove-from(stable): meeting/calling-composite */
+  ControlBarPeopleButton: <People20Regular />,
+  /* @conditional-compile-remove-from(stable): meeting/calling-composite */
```

Comment:
Further down in this file, this should have a `CallWithChatCompositeIcons` interface that 'picks' the ControlBarChatButtonActive, ControlBarChatButtonInactive etc.

And just noticing, CallCompositeIcons should not have
`| /* @conditional-compile-remove-from(stable) Chat_Notification_Icon */ 'ControlBarButtonBadgeIcon'` below

Reply:
(would be better as a separate PR)

Reply:
In fact, I'm doing some API changes, I'll update this when this PR goes in

Reply:
this was something I was wondering about when working on this, because we have Icon sets for the other two composites but we dont have a set listed for the new one.

Reply:
It's for contoso developers to easily have a list of icons they can override, though it's a shame we don't have any typeguarding when using icons to say - "hey! this icon isn't part of the CallWithChatCompoisteIcons interface! I'm, gonna fail your build"

We probably could but would have to define our own Chat/Call/CallWithChatCompositeIcon that wraps the fluent icon and only allows a subset of strings in the iconName that exist in the xCompositeIcon interface

Reply:
cool! I thought it might be something like this when intellisense didn't see the other icons right away!

Reply:
@JamesBurnside might be worth doing actually, I like that idea.
`<CallIcon iconName="something" />`
`<ChatIcon iconName="something" />`
`<CallWithChatIcon iconName="something" />`

Reply:
Actually doing it now! Along the lines of this:

![image](https://user-images.githubusercontent.com/2684369/155387611-5e2bebe4-cccc-4cd1-be5e-212430106d10.png)

Reply:
PR here: https://github.com/Azure/communication-ui-library/pull/1533